CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## [Unreleased]
 
+## [0.0.3] - 2024-09-05
+- Organisation service.
```

Comment:
Is this good?

Reply:
Yes, but you could write a sentence of the change itself. Maybe, if you have it, add a link to the PR